var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -12,14 +12,13 @@ class Proj(AutotoolsPackage):
     another. This includes cartographic projections as well as geodetic
     transformations."""
 
-    homepage = "https://proj4.org/"
-    url      = "http://download.osgeo.org/proj/proj-5.0.1.tar.gz"
-
-    # Version 6.0.0 deprecates the older API, and things which do
-    # not know this will fail to build. So, I recommend that we
-    # do not put in 6.x until you are ready to put in selective
-    # dependencies in all packages which depend on proj. I have
-    # done libgeotiff, but there are lots of others.
+    homepage = "https://proj.org/"
+    url      = "http://download.osgeo.org/proj/proj-6.1.0.tar.gz"
+
+    maintainers = ['adamjstewart']
+
+    version('6.1.0', sha256='676165c54319d2f03da4349cbd7344eb430b225fe867a90191d848dc64788008')
+    version('6.0.0', sha256='4510a2c1c8f9056374708a867c51b1192e8d6f9a5198dd320bf6a168e44a3657')
```

Comment:
Can you add back a brief comment noting that the API for 6.0.0 deprecates the previous API?